models/plan.py

```diff
@@ -86,7 +86,13 @@ def start_date(self) -> date:
         return self.__start_date  # Name mangling to access private field
 
     def __str__(self):
-        return f"Plan '{self.name}'"
+        if self.__is_closed:
+            return f"Plan: '{self.name}' (closed)\n" \
+                   f"Emergency Type: '{self.emergency.value.capitalize()}'\n" \
+                   f"Close Date: '{self.__close_date}'"
+        else:
+            return f"Plan '{self.name}'\n" \
+                   f"Emergency Type: '{self.emergency.value.capitalize()}'\n"
```

Comment:
Could probably avoid some duplicate code here but for now i think it's alright

Reply:
Alright, I'll leave it as it is then.